runtime/lua/vim/lsp/util.lua

```diff
@@ -748,8 +748,9 @@ do
     return popup_bufnr, winnr
   end
 
-  --- Saves the diagnostics (Diagnostic[]) into diagnostics_by_buf
-  --
+  --- Stores the diagnostics (Diagnostic[]) into |vim.lsp.util.diagnostics_by_buf|
```

Comment:
function tags always have parens. Vim will jump to it anyway, but the tag checker will complain (which adds noise to the tag checker, and eventually we might want to make that part of CI...)

```suggestion
  --- Stores the diagnostics (Diagnostic[]) into |vim.lsp.util.diagnostics_by_buf()|
```